### DISCUSSION TITLE
技术交流聊天群组

### DISCUSSION THREAD

**Discussion by liangjingkanji** [liangjingkanji/Net#192] [Announcements]
<img src="https://raw.githubusercontent.com/liangjingkanji/liangjingkanji/master/img/group-qrcode.png" width="50%"/>

README有群号, 有些时候可能会搜不到群号就请扫二维码

**Comment by yomigatz** [comment]
![image](https://github.com/liangjingkanji/Net/assets/30566819/8113a406-91cc-4431-9290-4ca150e79e83)
是这个群吗？感觉有点诡异
